ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -967,26 +1042,18 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
-    private lazy var tableViewIconTintColor: UIColor = GlobalTokens.neutralColor(.grey50)
-    private lazy var tableViewBackgroundColor: UIColor = view.fluentTheme.color(.background2)
-    private lazy var bottomBarBackgroundColor: UIColor = view.fluentTheme.color(.background2)
-
     private struct Constants {
```

Comment:
Shouldn't these also move to the tokenset extension?

Reply:
Some of these constants aren't really coming from the designs, but are there to just make things work, so I figured they were fine here, while the constants that are coming from design and using global tokens were moved to the tokenSet extension.